### ISSUE TITLE
Charged twice for subscription

### ISSUE THREAD

**Issue by Mevrhen** [sheibeck/fcs#165]
Hi there,

I was charged twice for the annual subscription. I sent an email asking for a refund for the second charge, but I never received a response. How can we resolve this issue?

**Comment by sheibeck** [comment]
Hi, so sorry about this. Something must be going on with stripe, your the
second contact me. I'll fix it up.

**Comment by sheibeck** [comment]
Can you tell me what email you were using so I can find your account?

**Comment by sheibeck** [comment]
Oh, I think I found your account and you are the person I already helped. Can you confirm that for me so I can close this if you're already taken care of.

**Comment by Mevrhen** [comment]
Hey, I'm so sorry! Same person. I was going to respond right away to
suggest that was the case but then had a family emergency and forgot to get
back to you.

Idk if I explained this,  but what happened was I signed up for the free
trial, but I was still unable to access those tools, so I tried again.
After the second time, I was able to access it. Idk if that helps or not.

Thanks for your help!

Jonathan
